src/Model/Item.php

```diff
@@ -2670,6 +2671,16 @@ private static function tagDeliver($uid, $item_id)
 			}
 		}
 
+		if (!$mention) {
+			$tags = Tag::getByURIId($item_id, [Tag::MENTION, Tag::EXCLUSIVE_MENTION]);
+			foreach ($tags as $tag) {
+				if (Strings::compareLink($link, $tag['url']) || Strings::compareLink($dlink, $tag['url'])) {
+					$mention = true;
+					Logger::log('mention found in tag: ' . $tag['url']);
```

Comment:
```suggestion
					DI::logger()->info('mention found in tag.', ['url' => $tag['url']]);
```

Reply:
I'm not able to determine the correct way to log the event and copied the previous few lines where the same mechanism was used. My apologies - could you please adjust as required?